### PR TITLE
feat: Added API to get namespace information

### PR DESCRIPTION
**Pull request by JigarJoshi**
## Describe your changes
Added API to get namespace information

## How best to test these changes
N/A

### CODE REVIEW

File: server/v1/management.proto [hunk 1]
```diff
@@ -185,6 +195,19 @@ service Management {
     };
   }
 
+  // Gets the namespace info for specific namespace
+  rpc GetNamespace(GetNamespaceRequest) returns
+      (GetNamespaceResponse) {
+    option (google.api.http) = {
+      post : "/v1/management/namespaces/get"
```
Comment (adilansari) [inline server/v1/management.proto:202]: `post` or `get` ?

Comment (adilansari) [reply server/v1/management.proto:202]: maybe `get: "/v1/management/namespaces/info"`

Comment (JigarJoshi) [reply server/v1/management.proto:202]: post. get doesn't work because it conflicts with other apis `get /v1/management/namespaces/list`

Comment (JigarJoshi) [reply server/v1/management.proto:202]: this is management api not to be used by all the users.

**Review by adilansari — APPROVED** [review]

**Pull request merged**